docs/DeveloperGuide.adoc

```diff
@@ -301,7 +301,15 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |delete a person |remove entries that I no longer need
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |user |view the full profile of a patient by searching for his/her name |see all details regarding a patient easily
+
+|`* * *` |user |record down details of each <<visitation,visitation>> |keep track of the patients situation
+
+|`* * *` |user |have daily reminders of important deadlines |keep track of appointments and visitations easily
+
+|`* * *` |user |set follow-up reminders |check in on my patients after some time or when their medication runs out
+
+|`* * *` |user |have <<user-defined-macros,user-defined macros>> |streamline my diagnosis documentation
 
 |`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
```

Comment:
Some comments on your User Stories:

1. The actors of your use cases are pretty general (user), maybe we could change them to something more descriptive and specific? (e.g. doctor)
1. Try to add more user stories! 😄 
1. Great that you linked specific definitions to your glossary! 👍